src/LazyOperators/lazy_operator.jl

```diff
@@ -9,9 +9,9 @@ and optionally:
 """
 abstract type AbstractLazy end
 
-function materialize(a::AbstractLazy, x)
-    error("`materialize` is not defined for:\n $(typeof(a)) \n and:\n $(typeof(x))")
-end
+# function materialize(a::AbstractLazy, x)
```

Comment:
can you remove it if it's not necessary anymore?

Reply:
done